src/modules/engage/components/FormBase.js

```diff
@@ -1,194 +1,45 @@
-import React, { Component } from 'react';
-import { Link } from 'react-router-dom';
+import { Component } from 'react';
 import PropTypes from 'prop-types';
-import { Wrapper } from 'modules/layout/components';
-import {
-  Button,
-  Icon,
-  FormControl,
-  ControlLabel,
-  FormGroup
-} from 'modules/common/components';
-import { EngageBox } from '../styles';
 
 const propTypes = {
+  brands: PropTypes.array,
+  users: PropTypes.array,
+  save: PropTypes.func,
   kind: PropTypes.string,
-  message: PropTypes.object,
-  save: PropTypes.func.isRequired,
-  users: PropTypes.array
+  message: PropTypes.object
 };
 
 class FormBase extends Component {
   constructor(props) {
     super(props);
 
-    const message = props.message || {};
-
-    this.state = { fromUser: message.fromUserId || '' };
-
-    // binds
-    this.generateDoc = this.generateDoc.bind(this);
+    this.state = {
+      a: 123
```

Comment:
ene haana ashiglagdaj bga state we? ashiglahgui bol ustgaad ashiglaj bga bol oilgomjtoi nereer bichih